bluemira/codes/plasmod/mapping.py

```diff
@@ -508,6 +516,8 @@ class Profiles(Model):
     # ############################
     # [-] plasma effective charge
     "Z_eff": ("Zeff", "amu"),  # TODO check dimensionless?
+    # [V] target loop voltage (if lower than -1e-3, ignored)-> plasma loop voltage
+    "v_burn": ("v_loop", "V"),
```

Comment:
We basically never want to prescribe the loop voltage to PLASMOD. I'm not entirely sure why this exists as a mode, but I'm certain we don't want to use it. Hence the move from an IN-OUT to an OUT only.